pycbc/inference/models/relbin.py

```diff
@@ -185,6 +190,16 @@ def __init__(
             if self.fid_params[k] == 'REPLACE':
                self.fid_params.pop(k)
 
+        # make sure fid_param has ['mode_array'] which means one
+        # needs to declare this in the model section
+        if 'is_hom' in self.static_params:
```

Comment:
Why do we need an option for this? Should it not simply be clear from the fact that you set the mode_array argument and have requested earth rotation that?

Reply:
The only reason was to extract the values of l,m, which are used further in evaluating the antenna pattern (f_l,m = 2/m*f_2,2)

Reply:
@SSingh087 You don't need an explicit option because you can just check that (1) they've set the mode array option and that it has a single value. That directly tells you that this must be true.